metaci/build/autoscaling.py

```diff
@@ -114,8 +115,23 @@ def scale(self):
             resp = requests.patch(
                 url, json={"quantity": self.target_workers}, headers=headers
             )
+            if resp.json() and resp.json().get("id") == "cannot_update_above_limit":
```

Comment:
You tested scaling above the hard limit (100). I wonder if we get a different message if there are x dynos running and we try to scale above (100 - x)?

Reply:
I did try that too. Updated to 99. Same message.

I'll double-check though.